### DISCUSSION TITLE
sync with PHP

### DISCUSSION THREAD

**Discussion by KoljaL** [tinyplex/tinybase#81] [General]
Hello TinyPeople,

I just found this repo and maybe it is a solution for a longlasting problem. 

But, is there a way to sync the tinybase data with a PHP server?

Have a nice day 😃

Kolja

**Comment by jamesgpearce** [comment]
We don't have an out of the box solution on the server here. But the remote persister saves and loads TinyBase data (in a simple way) to any server.

As long as you can issue and accept JSON in GET/POST, you should be good! Check out https://tinybase.org/api/persister-remote/functions/creation/createremotepersister/ and see if it helps.
